coalib/bears/Bear.py

```diff
@@ -68,6 +68,9 @@ def __init__(self,
             self.warn(error_string)
             raise RuntimeError(error_string)
 
+    def name(self):
```

Comment:
hey, can you make this a property? Then you can access it without the brackets `self.name` that'd be neat, woudln't it?